plugins/wysiwygarea/plugin.js

```diff
@@ -95,6 +95,40 @@
 
 					editor.editable( new framedWysiwyg( editor, iframe.$.contentWindow.document.body ) );
 					editor.setData( editor.getData( 1 ), callback );
+
+					iframe.$.onloadFromSetData = false;
+
+					editor.on( 'mode', backupOnLoad, { iframe: iframe, editor: editor, callback: callback } );
+				}
+
+				function backupOnLoad( evt ) {
```

Comment:
Please add a short comment with a ticket reference before this function following [the code guide](https://github.com/ckeditor/ckeditor4/blob/major/dev/docs/codestyle.md#210-comments). Also please adjust the other comments according to this (I mean mostly the point `Comment is a sentence - start it with capital leter, end with period.`).